### PR TITLE
 #196739: updated icon button styles per design specs

### PR DESCRIPTION
**Pull request by dubunin**
Met with design to hash out the missing redlines for the icon button. Updates icon button styles accordingly. 

Targeted issue: 196739

### CODE REVIEW

File: src/components/Button/Button.scss [hunk 1]
```diff
@@ -2,25 +2,36 @@
...
   }
-
+  .ms-Icon {
+    font-size: 16px;
+    padding: 8px;
+  }
   .ms-Button-icon {
     display: inline;
   }
+  &.is-disabled, &:disabled {
```
Comment (Jahnp) [inline src/components/Button/Button.scss]: nit: newline before block declarations

File: src/components/Button/Button.scss [hunk 2]
```diff
@@ -2,25 +2,36 @@
...
   }
 
   &:focus {
     background-color: transparent;
   }
-
+  .ms-Icon {
+    font-size: 16px;
+    padding: 8px;
+  }
```
Comment (Jahnp) [inline src/components/Button/Button.scss]: nit: newline before block declarations

**Comment by Jahnp** [comment]
Approved 🍊 

With small stylistic suggestions. Looks great!

File: src/components/Button/Button.scss [hunk 3]
```diff
@@ -2,25 +2,36 @@
...
   border: 0;
 
-  &:hover {
+  &:hover,
+  &:active {
     background-color: transparent;
     border-color: transparent;
     outline-color: transparent;
+    color: $ms-color-neutralPrimary;
   }
```
Comment (cschlechty) [inline src/components/Button/Button.scss]: Please verify this still looks like a button or how you'd expect in high contrast.

Comment (dubunin) [inline src/components/Button/Button.scss]: Double checked with design to make sure this is the desired look. Also, worked with design to figure out high contrast colors for icon buttons. Decided best to use colors that match up with link colors in high contrast. Per design no hover color change necessary in high contrast. Cursor changing to pointer is change enough.

**Pull request merged**